utils/ALA_dialogues.cfg

```diff
@@ -365,11 +365,11 @@
     {MOVE_UNIT (id=Divaid,Croquet) 8 13}
     [message]
         speaker=Glauco
-        message= _ "What's the situation, "+$trapper_name+"?"
+        message= _ "What's the situation?"
```

Comment:
I guess this part is to make it easier for translation? OK if that's the case

Reply:
Yes, it is always easier without variable. And here it's clear who is being addressed.

Reply:
ok